CHANGES.md

```diff
@@ -22,6 +22,7 @@
 -   Use certifi to set `GDAL_CURL_CA_BUNDLE` / `PROJ_CURL_CA_BUNDLE` defaults (#97)
 -   automatically detect driver for `.geojson`, `.geojsonl` and `.geojsons` files (#101)
 -   read DateTime fields with millisecond accuracy (#111)
+-   support writing object columns with np.nan values (#60)
```

Comment:
```suggestion
-   support writing object columns with np.nan values (#118)
```

(I've generally been putting the PRs instead of originating issues in the changelog, but then this requires first creating the PR before then adding an etry here...)